doc/guide/cockpit-session.xml

```diff
@@ -58,6 +58,32 @@ promise.then(user => { ... });
     </variablelist>
 
     <para>Returns a promise that completes once the user information is available.</para>
+    <warning>
+	    <para>
+		    <code>cockpit.user()</code> is soft-deprecated since Cockpit 336, <link
+		    linkend="cockpit-init">cockpit.init()</link> can be used
```

Comment:
This documentation is inaccurate.  It's `cockpit.info` that has that information, but you indeed need to call `.init()` first.  Needs rewording.